hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -56,17 +58,23 @@ if {$FOUNDRY_ROOT != ""} {
 set ri_create_outputs_in_create_env true
 set ri_print_module_nand2_counts true
 # enable analysis of large arrays
-set ri_max_total_range_bits 100000
+set ri_max_total_range_bits 20
+set ri_prefer_liberty true
+
+############################
+## Apply Liberty File ##
+
+read_liberty $AST_LIB -db_dir ./RI_compiled_libs/verix_2019_a_p27_2_2022_03_20
```

Comment:
Thanks for adding the AST_LIB, that part LGTM!

For the second part, if this is just scratch storage and the name can be arbitrary, can we name this differently and remove the tool version from the name? if not explicitly needed, it would be good to avoid checking explicit tool version names...

We could name it e.g. `./RI_compiled_libs/verix_libs`...